DistFiles/factoryCollections/Sample Shells/Vaccinations/previewMode.css

```diff
@@ -1,4 +1,4 @@
-﻿@media screen {
+@media screen {
```

Comment:
Every time I run Bloom it updates that file. I've reverted it about 100 times, and decided to just commit it so it stops pestering me.